### PR TITLE
Delete edit reminders notifications

### PR DESCRIPTION
**Pull request by DONOTFORGET-BYTE**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

File: django_project/frontend/templates/notifications.html [hunk 1]
```diff
@@ -373,7 +298,7 @@ <h2 id="notificationTitle"></h2>
     }
 
     .pagination a.page-link {
-        color: green;
+        color: #70B276;
```
Comment (dimasciput) [inline django_project/frontend/templates/notifications.html:301]: @DONOTFORGET-BYTE please use css variable instead.

**Review by dimasciput — CHANGES_REQUESTED** [review]

File: django_project/frontend/templates/notifications.html (hunk 1 above)
Comment (dimasciput) [reply django_project/frontend/templates/notifications.html:301]: for example

```
:root {
  --main-green: #70B276;
}

.pagination a.page-link {
  color: var(--main-green);
}
```

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 19 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 20 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 22 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 21 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 24 min

**Pull request merged**